src/main/java/org/terasology/MooConstants.java

```diff
@@ -57,7 +57,7 @@
     public static final String GUARD_BUILDING_COMMAND_UI_ID = "guardBuildingCommand";
 
     // Crop to be placed after plant task
-    public static final String OREON_CROP_0_BLOCK = "MasterOfOreon:OreonPlant0";
+    public static final String OREON_CROP_0_BLOCK = "MasterOfOreon:FudgeFlower0";
```

Comment:
This changes the crop which is placed after the plant task. So having different types of crops seems moot.
I believe we should change the implementation to incorporate an extra screen during task assignment. After the player selects a plant task a screen pops up with the types of available crops.
Let me know if you can think of another user friendly way to handle this :).

Reply:
I'm interested in this new task; I think the extra screen is a good way to handle it! 

I brainstormed a couple of ideas for the screen too:
- Photo preview of final plant form or animation of it growing (like a GIF of it growing) 
----- I haven't seen any photos on the menu so I don't know if this is possible, but if it is I think it would definitely be cool
- Option for a random plant in addition to the other plant options

I designed two basic mockups of the screens too, attached to this comment:
![PlantSelectOutline](https://user-images.githubusercontent.com/23141240/72648241-8f21b180-3948-11ea-8de9-d07b053b7f7c.png)

Would you like me to code this screen option into MOO?

Reply:
+1 on the new task. I suggest separating it into two different PRs: the current one (for the task already approved, reverting the changes to OREON_CROP_0_BLOCK) and a new one for the derived task.

Reply:
@casals Reverted :)

Reply:
Thanks! @Naman-sopho I'm creating a draft task called MOO Crops (III) to accommodate the new screen - but leaving the conversation/MR for you in the case there's something else to tackle here.

Reply:
@casals Thanks for the task write up.

@HannahGuo The mock up for the screen looks good. You can claim the [MOO Crops{III)](https://codein.withgoogle.com/dashboard/tasks/4839611407269888/) task. You can update this PR once you are done. You can join the Discord channel and ping me if you have any questions.